packages/meditrak-server/src/routes/importSurveys/Validator/ConfigValidator/ConfigValidator.js

```diff
@@ -32,8 +33,10 @@ export class ConfigValidator extends BaseValidator {
         return EntityConfigValidator;
       case CODE_GENERATOR:
         return CodeGeneratorConfigValidator;
-      case CALCULATED:
-        return CalculatedConfigValidator;
+      case ARITHMETIC:
+        return ArithmeticConfigValidator;
+      case CONDITION:
+        return ConditionConfigValidator;
```

Comment:
The logic of Calculated Question is getting a bit complex and I felt that there were a lot of switch cases to process things differently between `arithmetic` and `condition` type (and there's little common between them). So I decided to split CalculatedQuestion into ArithmeticQuestion and ConditionQuestion (which I remember you did suggest before).